### ISSUE TITLE
Template for category

### ISSUE THREAD

**Issue by hchautrung** [ErnstHaagsman/jsdoc-plugins#2]
Hi,

The category plugin is very useful but there is no custom template to show the category. And I read the source code categories.js What is the purpose of the function **processCategories**. The function has been used, is it?

Thanks,
Trung,

**Comment by prototype23** [comment]
Agreed, nice plugin but is not completed yet? I did the setup, Docs are generated but a cannot see any category. I'm using the default jsDoc3 template.

**Comment by ErnstHaagsman** [comment]
I wrote this plugin for a company I worked for a couple years ago. We had a custom template for our docs, and I don't have access to those sources anymore. 

This plugin is definitely not supported by the default template, you'd need to write some code to handle the category information in your template.

**Comment by prototype23** [comment]
oh, thanks for the clarification :)

**Comment by hchautrung** [comment]
Hi all,

Due to requirements of my startup project. I have implemented a template based on minami can support.
- Category
- RestAPI tags: @api, @apiparam, @apiquery etc
- Expand/Collapse items.
- Search
- ect.

If you think it can help please let me know I will setup on github

![docstruct](https://cloud.githubusercontent.com/assets/2569420/20166321/e801ca04-a74e-11e6-9de0-33d614670e5e.png)

**Comment by anius** [comment]
@hugohuynh Could you share your template on github? My project requires similar structure and your template looks really great. Thanks.

**Comment by magiclen** [comment]
I think it helps a lot. The default template is not useful.

**Comment by suryanaik** [comment]
@hugohuynh Can you put the template on Github so that many of us can benefit from it.
